exercises/word-count/Phrase.groovy

```diff
@@ -0,0 +1,10 @@
+class Phrase {
```

Comment:
We don't have to do it in this pull request, but we should make the file names match the name of the exercise to make it easier to do automation (like refer to the files in the README generation).